equinix/resource_network_device_link.go

```diff
@@ -147,7 +173,15 @@ func createNetworkDeviceLinkResourceSchema() map[string]*schema.Schema {
 				Schema: createNetworkDeviceLinkConnectionResourceSchema(),
 			},
 			Set:         networkDeviceLinkConnectionHash,
-			Description: networkDeviceLinkSchemaNames["Links"],
+			Description: networkDeviceLinkDescriptions["Links"],
```

Comment:
`link` is deprecated in the markdown. I assume we want to actually deprecate the field here.
https://developer.hashicorp.com/terraform/plugin/sdkv2/best-practices/deprecations#provider-attribute-removal